package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-cli",
-  "version": "2.1.2-alpha10",
```

Comment:
Shouldn't this be `2.1.2-alpha11` ?

Reply:
Alpha is stable enough to release a new version